crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -119,6 +119,20 @@ impl Mesh {
             );
         }
 
+        // validate attributes
+        match attribute.id {
+            id if id == Self::ATTRIBUTE_JOINT_WEIGHT.id => {
```

Comment:
Looks like the `match` is unused. (could just be an ordinary if statement)

Reply:
Hmm, this would be nicer as a match statement when other attributes need validation, but it doesn't seem possible :(

Reply:
> this would be nicer as a match statement when other attributes need validation

i'm not sure why but it felt more natural to make it a match, i guess this is what i was thinking.  but i'll amend, we can always change it if we need more in the future.